action/config/generate.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package config
+
+import (
+	"path/filepath"
+
+	"github.com/spf13/afero"
+
+	yaml "gopkg.in/yaml.v2"
+)
+
+// create filesystem based on the operating system
+var appFS = afero.NewOsFs()
+
+// Generate produces a pipeline based off the provided configuration.
+func (c *Config) Generate() error {
```

Comment:
This is where we add the new `Generate()` function.

The purpose of this function is to generate a valid Vela configuration file for making requests.